README.md

```diff
@@ -2,74 +2,17 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Hello-Microverse!
```

Comment:
Kindly work on these changes:
- Your readme file is missing general information about the creator, descriptive information about on how to download and install your project locally, once installed how to run it, which requirements are needed before to use it probably in this case isn't needed to have information about testing and deployment.
- You can use [this template.](https://github.com/microverseinc/readme-template)